packages/cuba-react-ui/src/ui/paging/Paging.tsx

```diff
@@ -0,0 +1,123 @@
+import {PaginationConfig} from "antd/lib/table";
+import {DataCollectionStore} from "@cuba-platform/react-core";
+import * as QueryString from "querystring";
+import React from "react";
+import {Pagination} from "antd";
+import {toJS} from "mobx";
+
+type Props = {
+  paginationConfig: PaginationConfig;
+  total?: number,
+  onPagingChange: (current: number, pageSize?: number) => void;
+}
+
+/**
+ * Paging defaults
+ */
+export const defaultPagingConfig: PaginationConfig = {
+  current: 1,
+  pageSize: 10,
+  defaultPageSize: 10,
+  showSizeChanger: true,
+  pageSizeOptions: ['10', '20', '30']
+};
+
+/**
+ * Ant Design Pagination component adopted to our List and DataCollectionStore
+ */
+export class Paging extends React.Component<Props> {
+
+  render() {
+    const {paginationConfig, total} = this.props;
+
+    return (
+      <Pagination
+        {...paginationConfig}
```

Comment:
I wonder if we should introduce our component at all

Reply:
I vote for introduce component. It's help us to support generated code (and ejs templates) more compact and readable. Also we get rid of copy paste in each generated list.